### ISSUE TITLE
Moto 1.3.4?

### ISSUE THREAD

**Issue by haydenth** [localstack/localstack#889]
Hello friends!

I see in the requirements.txt file that this library is using

```moto-ext==1.3.3```

It looks like two days ago, moto released version 1.3.4 - would it be possible to move to this newer moto version? I'm looking for functionality around apigateway's get-api-keys call which is in this newer moto release.

**Comment by mberlanda** [comment]
Hey @haydenth, could you write a short test or explain how to reproduce what you are attempting to achieve?

I would be glad to create a PR to update this dependency tomorrow, but I would make sure first that it will respond to you need

**Comment by mberlanda** [comment]
It seems that the latest version of `moto` is not yet available in pip
```
$ pip install moto-ext==1.3.4
Collecting moto-ext==1.3.4
  Could not find a version that satisfies the requirement moto-ext==1.3.4 (from versions: 0.4.26, 0.4.32, 0.4.32.1, 0.4.32.2, 1.0.0.1, 1.0.0.2, 1.0.0.3, 1.0.0.4, 1.0.0.5, 1.0.1.1, 1.0.1.2, 1.0.1.3, 1.0.1.4, 1.0.1.5, 1.0.1.6, 1.0.1.7, 1.0.5, 1.1.1, 1.1.5, 1.1.21, 1.1.24, 1.1.25, 1.1.25.1, 1.1.25.2, 1.1.25.3, 1.2.0, 1.3.3)
No matching distribution found for moto-ext==1.3.4
```

**Comment by whummer** [comment]
Fixed in #890
